README.md

```diff
@@ -70,30 +70,34 @@ See also [About Us](http://px4.io/about-us/#development_team) (px4.io) and the [
 ## Supported Hardware
 
 This repository contains code supporting these boards:
-  * [Snapdragon Flight](https://docs.px4.io/en/flight_controller/snapdragon_flight.html)
-  * [Intel Aero](https://docs.px4.io/en/flight_controller/intel_aero.html)
-  * [Raspberry PI with Navio 2](https://docs.px4.io/en/flight_controller/raspberry_pi_navio2.html)
-  * [Parrot Bebop 2](https://dev.px4.io/en/advanced/parrot_bebop.html)
+  * [Snapdragon Flight](https://docs.px4.io/master/en/flight_controller/snapdragon_flight.html)
```

Comment:
@hamishwillee Shouldn't master be the default documentation version if we anyways link it everywhere?
I still see people struggle to find things that were changed in the documentation since 1.9 and even I sometimes get on the 1.9 page and there's no button to switch to the master version without going to the landing page again. Changing the browser URL manually works.

Reply:
Master is the default if you go to the root https://docs.px4.io/ or https://docs.px4.io/en/. But if you got to an existing bookmarked link then this goes to 1.9 version. That seemed reasonable as the bookmarks are then all fixed - ie I don't need to regenerate the manual links again.
In future whatever specific version people actually bookmark will be where the link goes.

I wasn't entirely sure whether root should go to master or somewhere else. I tend to work in master, but others work in specific versions or want to go to "current stable".